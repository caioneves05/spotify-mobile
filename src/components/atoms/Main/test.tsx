import renderer from 'react-test-renderer';

import { screen } from '@testing-library/react-native';

import { renderWithTheme } from 'utils/tests/helpers';

import { Main } from '.';

describe('<Main />', () => {
  it('should render the heading', () => {
    renderWithTheme(<Main />);

    const container = renderer.create(<Main />).toJSON();

    expect(screen.getByText(/main/i)).toBeDefined();

    expect(container).toMatchSnapshot();
  });
});
