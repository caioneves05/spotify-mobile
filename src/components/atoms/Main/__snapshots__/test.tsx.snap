// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "#000",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "#fff",
        },
      ]
    }
    testID="main"
  >
    Main
  </Text>
</View>
`;
